app/scripts/modules/docker/tsconfig.json

```diff
@@ -1,18 +1,13 @@
 {
-  "extends": "../../../../tsconfig.json",
+  "extends": "../../../../tsconfig.base.json",
   "compilerOptions": {
     "baseUrl": "./src",
-    "declaration": true,
-    "declarationDir": "lib",
-    "outDir": "lib",
-    "rootDir": "./src",
-    "typeRoots": ["../../../../node_modules/@types"],
+    "outDir": "dist",
     "paths": {
-      "@spinnaker/core": ["../../core/lib"],
-      "core/*": ["core/src/*"]
-    }
+      "core/*": ["../node_modules/@spinnaker/core/lib/*"]
```

Comment:
Are we planning to remove these imports as well?

Reply:
I also thought `core/lib` will be removed

Reply:
> Are we planning to remove these imports as well?

It is not planned at the moment. But can be handled via a codemod at the some point after the current migration is done.